recipe/meta.yaml

```
@@ -2,6 +2,8 @@
 {% set version = "12.5.40" %}
 {% set cuda_version = "12.5" %}
 {% set cuda_version_next_major = (cuda_version.split(".")[0]|int + 1)|string + ".0a0" %}
+{% set gcc_lower_bound = ">=6" %}
+{% set gcc_upper_bound = "<14.0a0" %}
```

Some other things we could consider 

We could make this one spec like

```suggestion
{% set gcc_constraint = ">=6,<14.0a0" %}
```

Or we could move the comparisons into the lines below

```suggestion
{% set gcc_lower_bound = "6" %}
{% set gcc_upper_bound = "14.0a0" %}
```

---

I considered these but I like the way it is. We need to separate lower and upper because there are some parts where we only pin the lower bound for libgcc-ng. I like keeping the sign as part of the specifier because it allows us to change between > and >= without further changes.

---

The first one would allow us to exclude problematic versions within the bounds as well

```
gcc >=6,!=11.0.0,<14.0.0a0
```

---

Okay. I updated in 2c2f162.